spec/controllers/graphql/queries/freaks_spec.rb

```diff
@@ -13,11 +13,17 @@ module Graphql
     end
 
     before do
-      create(:freak, first_name: 'Ion')
-      create(:freak, first_name: 'Gheorghe')
+      create(:role, id: 1, name: 'Founder')
+      create(:freak, role_id: 1)
```

Comment:
A freak already has an associated role in the factory, so there's no need to re-create a new role.